group_layers/group-layers/README.md

```diff
@@ -2,22 +2,27 @@
 
 Group a collection of layers together and toggle their visibility as a group.
 
-![](GroupLayers.png)
+![Image of group layers](GroupLayers.png)
 
 ## Use case
 
 Group layers communicate to the user that layers are related and can be managed together.
+
 In a land development project, you might group layers according to the phase of development.
 
 ## How to use the sample
 
-The layers in the map will be displayed in a table of contents. Toggle the checkbox next to a layer's name to change its visibility.
+The layers in the map will be displayed in a table of contents. Toggle the checkbox next to a layer's name to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
 
 ## How it works
 
 1. Create an empty `GroupLayer`.
 2. Add a child layer to the group layer's layers collection.
-3. To toggle the visibility of the group, simply change the group layer's visibility property.
+3. To build the table of contents:
```

Comment:
we do this quite differently in Java and the only step we follow here is checking if `isShowChildrenInLegend` is true. I'm not sure how best to write up our steps when a lot of it is Java logic to recursively populate the table of contents with layer names?

Reply:
Thanks for pointing this out. Considering most of the building of the 'tree' is UI focused, I've reverted back to the old java 'how it works' section, with a few additions. Looks like the main point is to show how to use the `GroupLayer` class and toggling visibility, so we don't need to do much more than that. What do you think?